modsulator.gemspec

```diff
@@ -16,10 +16,13 @@ Gem::Specification.new do |s|
   s.platform    =  Gem::Platform::RUBY
   s.executables << 'modsulator'
 
+  s.required_ruby_version = '~> 3.0'
```

Comment:
We haven't pushed this gem for 3 years. Does it still need to be a gem?

Reply:
Dunno.  Arcadia or someone(s) in metadata use it and I am not sure how they are using it.

Reply:
It sounds like they are not using it as a gem.

Reply:
See https://github.com/sul-dlss/modsulator-app-rails/pull/245